### ISSUE TITLE
HUD

### ISSUE THREAD

**Issue by Monofoot** [PlaceholderGames/2016-Team2Thirds#15]
Connor has informed me that the menu currently allows the selection of a multiplayer game. I've worked on elements of the HUD and it's looking good, however I'm waiting on a commit from Kieran until I can merge the branch.
Currently the main menu and the skill bar at the bottom of the screen for the third person character has been worked on.

**Comment by Monofoot** [comment]
*Waiting on Kieran because of the interaction with the traps.

**Comment by Monofoot** [comment]
![image](https://cloud.githubusercontent.com/assets/22819110/20612335/bcb4bc5e-b2b0-11e6-84b9-04203b7d7630.png)

Minimap is working and follows the player. The red dot is a throbber and 'throbs', which is a good starting point for the second part of the minimap which is detecting the enemy player (through 'pings', or throbs).

**Comment by Monofoot** [comment]
![image](https://cloud.githubusercontent.com/assets/22819110/20612362/fd4ab1b0-b2b0-11e6-936a-2f3c67449d76.png)
Better look at the minimap (I got lazy and copy pasted the screenshot directly....)

**Comment by Monofoot** [comment]
I'm going to close this issue and move it into a separate issue. "HUD" is too vague and there are too many aspects to cover here.
To update: I'm going to start committing HUD changes to the "TestTrap", as this gives me access to four traps for me to create a GUI around. After further meeting with the team we can decide where the other GUI elements need to be.
